src/tests/test_data.py

```diff
@@ -17,6 +17,20 @@ def get_all_test_models():
     models = get_other_rec_test_models()
     return models
 
+def rhythmic_value_to_duration(rhythmic_value):
```

Comment:
Is can be in utils.constants

Reply:
I disagree, that would mean that we connect output format for frontend with input format for tests

Reply:
Yeah, those are the same formats

Reply:
Done